c/src/ml-api-service.c

```diff
@@ -67,6 +68,12 @@ _ml_service_set_information_internal (ml_service_s * mls, const char *name,
     case ML_SERVICE_TYPE_EXTENSION:
       status = ml_service_extension_set_information (mls, name, value);
       break;
+    case ML_SERVICE_TYPE_REMOTE:
+    {
+      if (g_ascii_strcasecmp (value, "path") == 0)
```

Comment:
I recommend adding _set_information() function and comparing 'name' for later work. Now offloading feature handles path only, but we may need another info.